README.md

```diff
@@ -441,7 +441,7 @@ Options:
                                 be given multiple times), alias: environment_variables, format: /[\w\-]+=.+/,
                                 note: Can be encrypted for added security)
   --runtime NAME                Lambda runtime to use (type: string, default: nodejs10.x, known values:
-                                nodejs12.x, nodejs10.x, python3.8, python3.7, python3.6, python2.7, ruby2.7,
+                                nodejs14.x nodejs12.x, nodejs10.x, python3.8, python3.7, python3.6, python2.7, ruby2.7,
```

Comment:
According to [Contributing to Dpl](https://github.com/travis-ci/dpl/blob/master/CONTRIBUTING.md#updating-the-readme), you should not update `README.md` directly.